std_msgs/Header header

#Global Position
float64 global_latitude
float64 global_longitude
float64 global_altitude

#Local Position
float64 local_x
float64 local_y
float64 local_z
